ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -604,7 +823,7 @@ class EclCmdRun : public EclCmdWithEclTarget
             req->setCluster(wuCluster.str());
         else if (optTargetCluster.length())
             req->setCluster(optTargetCluster.get());
-        req->setWait((int)optWaitTime);
+        req->setWait((int)(optPoll ? 0 : optWaitTime));
```

Comment:
Yes before I made the polling an explicit user option that was my intent.  No reason not do do it even now.  I'll make the change.  I may make the initial wait even longer, say 5 or 10 secs?